scripts/examples/simple_async_tcp_client.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# scripts/examples/simple_async_tcp_client.py
```

Comment:
Nitpick: name of this file should probably be `simple_asyncio_tcp_client.py` (in stead of async)

Reply:
Ok.
Have in mind that although I called the branch *asyncio-client*, in fact, the entire PR is async library agnostic.